src/stash/src/postgres.rs

```diff
@@ -76,12 +78,59 @@ struct PreparedStatements {
     iter: Statement,
     seal: Statement,
     compact: Statement,
-    consolidate_delete: Statement,
-    consolidate_consolidate: Statement,
-    consolidate_cleanup: Statement,
     update_many: Statement,
 }
 
+impl PreparedStatements {
+    async fn from(client: &Client) -> Result<Self, StashError> {
```

Comment:
You might be able to implement the `TryFrom<&Client>` trait here. Though it's not really important.

Reply:
I agree with both sentences and just kinda don't care too much.